### DISCUSSION TITLE
Importing from not bundled module

### DISCUSSION THREAD

**Discussion by sheremet-va** [cypress-io/cypress#17019] [General]
At our company we have a repository with front end packages (not as a monorepo). We are using typescript and do not transform them to js output to get advantage from using tree-shaking. It is working fine using Vite (so, no Webpack - only rollup + esbuild) in dev and production environments, but if I import something from a package, tests fail with an error:

```
Error: Webpack Compilation Error
./node_modules/@company/shared/src/libs/colors.ts 10:39
Module parse failed: Unexpected token (10:39)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
export function getContrastYIQ(hexColor: string, darkColor = '#000000', lightColor = '#ffffff') {
```

It there a way to configure it, so Webpack inside Cypress can understand it?

**Comment by esskar** [comment]
@sheremet-va We have a similar setup and I run in the same issue; have you found a way around?
